src/goto-programs/goto_inline.cpp

```diff
@@ -153,6 +174,9 @@ void goto_partial_inline(
     if(gf_entry.first == goto_functions.entry_point())
     {
       // Don't inline any function calls made from the _start function.
+      // This is likely so that the convention of __CPROVER_start
```

Comment:
I think we could be more firm here, let's just state that this is the reason. No need to speculate in the comment.

Reply:
I have made the wording more definite.